retrofit/src/main/java/org/devfleet/crest/retrofit/CrestServiceImpl.java

```diff
@@ -269,6 +276,64 @@ private boolean setWaypoints(final List<CrestWaypoint> waypoints, final boolean
         }
     }
 
+    @Override
+    public List<CrestMarketOrder> getMarketOrders(final long regionId, final String orderType, final long itemId) {
+        try {
+            CrestDictionary<CrestMarketOrder> dictionary;
+
+            // TODO: Change this so that it pulls the url from the root CREST
+            // endpoint
+            final String typePath = href("inventory/types") + itemId + "/";
+
+            dictionary = this.publicCrest().getMarketOrders(regionId, orderType, typePath).execute().body();
+
+            return dictionary.getItems();
+        } catch (IOException e) {
+            LOG.error(e.getLocalizedMessage(), e);
+            throw new IllegalStateException(e.getLocalizedMessage(), e);
+        }
+    }
+
+    @Override
+    public List<CrestMarketBulkOrder> getAllOrders(final long regionId) {
+        try {
+            final List<CrestMarketBulkOrder> returned = new ArrayList<>();
+            CrestDictionary<CrestMarketBulkOrder> dictionary;
+            int page = 0;
+            do {
+                page = page + 1;
+                dictionary = this.publicCrest().getAllMarketOrders(regionId, page).execute().body();
+                if (null == dictionary) {
+                    LOG.error("getAllOrders: null dictionnary {}, {}", regionId);
+                    break;
+                }
+                returned.addAll(dictionary.getItems());
+            } while (dictionary.getPageCount() > page);
+            return returned;
+        } catch (IOException e) {
+            LOG.error(e.getLocalizedMessage(), e);
+            throw new IllegalStateException(e.getLocalizedMessage(), e);
+        }
+    }
+
+    @Override
+    public void getAllOrdersAsync(final long regionId, final Callback<CrestDictionary<CrestMarketBulkOrder>> cb) {
```

Comment:
I don't really want this layer to provide async features. The callers of the interface should do it if they feel like it.

The way the interface is typically used is using RxJava, which will do a better job of managing threads. But again, clients of the interface should deal with async, not us. 

Please remove this method